packages/contentstack-seed/src/typings/@contentstack/cli-utilities.d.ts

```diff
@@ -1,5 +1,43 @@
 declare module '@contentstack/cli-utilities' {
```

Comment:
do we need to declare the types in this package

Reply:
removed type definition module from all modules and added type index in the package.json